riscv/insns/vaesef_vv.h

```diff
@@ -0,0 +1,61 @@
+// vaesef.vv vd, vs2
+
+#include "zvkned_ext_macros.h"
+#include "zvk_ext_macros.h"
+
+// Uncomment to enble debug logging of invocations of this instruction.
```

Comment:
```suggestion
// Uncomment to enable debug logging of invocations of this instruction.
```

Reply:
Done here and for all the cut/pasted instances. Thanks for catching this.